client/src/app/pages/dashboard/components/task-item/task-item-in-progress/task-item-in-progress.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="root">
```

Comment:
We should use one component. And one more thing about name - it should not be like 'in progress', because we even don't know it (as a rule this is planned task). SO it will be better to use name just like task-item

Reply:
It would be great if I could use the same component. However, a completed task item does not have a checkbox. In the beginning you said, if i am not mistaken, that there is not need for checkboxes for completed task items. This is why i had to create two components